package.json

```diff
@@ -77,13 +81,15 @@
     "katex": "^0.10.0",
     "lodash.debounce": "^4.0.3",
     "lodash.get": "^4.3.0",
+    "loose-envify": "^1.4.0",
     "mkdirp": "^0.5.1",
     "mocha": "^6.0.0",
     "ng-tags-input": "^3.1.1",
     "node-uuid": "^1.4.3",
     "npm-packlist": "^1.1.12",
     "postcss": "^7.0.13",
     "postcss-url": "^8.0.0",
+    "preact": "10.0.0-alpha.2",
```

Comment:
We're using the pre-release version here because it supports [fragments](https://reactjs.org/docs/react-api.html#fragments) and the [modern "Context" API](https://reactjs.org/docs/context.html#when-to-use-context) which are both going to be useful for us.